themes/finna2/less/finna/result-sidebar.less

```diff
@@ -75,7 +75,8 @@
 }
 
 .sidebar :not(.recordProvidedBy) h1,
-.sidebar > h1 {
+.sidebar > h1,
+.sidebar > h2.margin-bottom {
```

Comment:
Onko sidebar:ssa joku muu h2, johon ei saa osua? Tai voiko laittaa `:first-of-type`? Toi margin-bottom -luokka on vaan vähän kamala.. :)

Reply:
Pasi sanoi tsekkaavansa tätä.

Reply:
Siellä ei pitäisi olla muita h2-elementtejä. Päivitin tämän koskemaan kaikki sidebarin h2-elementtejä.